レビュー用PR_2023_03_24に関しての修正

sai-gillingham opened the pull request:
https://github.com/EC-CUBE/TwoFactorAuthCustomer42/pull/3#discussion_r1147157803
について、他の-`>findOneBy([〇〇]) === null`箇所の代わりに`count`関数を利用するように更新しました。

https://github.com/EC-CUBE/TwoFactorAuthCustomer42/pull/3#issuecomment-1482332934
について、スロットリングパラメータはルーティング名と合わせました。

sai-gillingham committed:
findOneByではなくcountを利用するように修正しました。
https://github.com/EC-CUBE/TwoFactorAuthCustomer42/pull/3#discussion_r1147157803

sai-gillingham committed:
スロットリングパラメータはルーティング名と合わせました。
https://github.com/EC-CUBE/TwoFactorAuthCustomer42/pull/3#issuecomment-1482332934

kiy0taka reviewed (changes requested):
`app_challenge_request ` もルーティング名と合わせるようにお願いします。

sai-gillingham committed:
スロットリングパラメータミスの修正
https://github.com/EC-CUBE/TwoFactorAuthCustomer42/pull/3#issuecomment-1482332934

sai-gillingham commented:
レビューして頂いてありがとうございます。


すみません、確かにルーティング名に合わせるミスがありました。
先ほどapp_challenge_requestもルーティング名と合わせましたのでご確認のお願いを頂けますか？
